refinedstorage2-platform-common/src/main/java/com/refinedmods/refinedstorage2/platform/common/screen/grid/GridScreen.java

```diff
@@ -409,6 +409,14 @@ private boolean isModifiedJustNow(final LastModified lastModified) {
             && lastModified.amount() <= MODIFIED_JUST_NOW_MAX_SECONDS;
     }
 
+    @Nullable
+    public GridResource getCurrentlyHoveredResource() {
```

Comment:
```suggestion
    public GridResource getHoveredResource() {
```

(`currently` is unneeded information)